stable/mission-control/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # JFrog Mission-Control Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [0.7.0] - Nov 16, 2018
+* Updated Mission-Control version to 3.3.0
+* Remove usage of certificates for internal communication
+
```

Comment:
Please add upgrade instructions from 3.2.0 (if needed)

Reply:
No extra steps needed for this upgrade.